### ISSUE TITLE
Spacebar  button is not working

### ISSUE THREAD

**Issue by Furqat-Abduvosiqov** [armoar334/linecomp#10]
After installation in the terminal, the spacebar button stopped working.

**Comment by armoar334** [comment]
When you press space, is there an error that flashes, or does it just not register at all?
Are you able to run a single command (i.e ls)?

**Comment by Furqat-Abduvosiqov** [comment]
Yes i can run single command like ls .But when i press sapce bar it is not
typing nothing.

**Comment by armoar334** [comment]
Thats very odd, I cant seem to replicate it locally. It might be something in your inputrc that is messing with the spacebar, as that would be the only external thing that would mess with the any of the keybinds
